config.md

```diff
@@ -855,8 +823,7 @@ Here is a full example `config.json` for reference.
 [selinux]:http://selinuxproject.org/page/Main_Page
 [no-new-privs]: https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt
 [procfs_2]: https://www.kernel.org/doc/Documentation/filesystems/proc.txt
-[semver-v2.0.0]: http://semver.org/spec/v2.0.0.html
-[go-environment]: https://golang.org/doc/install/source#environment
+[semver-v3.0.0]: http://semver.org/spec/v2.0.0.html
```

Comment:
I don't think you want to bump `semver-v2.0.0` → `semver-v3.0.0` (especially since 2.0.0 is still the most recent version ;).